### PR TITLE
Add Piped

### LINKED ISSUE #9283: Piped
### Brand Name

Piped

### Website

https://piped.video/

### Popularity Metric

Github stars: 5.1k stars. See https://github.com/TeamPiped/Piped

### Official Resources for Icon and Color

svg: https://piped.video/img/icons/logo.svg
color: #d43225 - eyedropped from the svg

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by SebiTalent04**
piped

**Issue:** closes #9283

**Similarweb rank:** 58,980
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Not sure if I would need to add some gaps where the holes are.

### CODE REVIEW

**Comment by adamrusted** [comment]
Ooh this is a fun one!
Personally, I'd add a gap between the two areas of the pipe, as below
image

What would other @simple-icons/icon-review folk recommend?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9370,6 +9370,11 @@
             "hex": "1A1928",
             "source": "https://www.pioneerdj.com"
         },
+        {
+            "title": "Piped",
+            "hex": "D43225",
+            "source": "https://piped.video"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:9376]:
```suggestion
            "source": "https://github.com/TeamPiped/Piped/blob/615f1e64d72181f11d7a002b63a4d26028619230/public/img/icons/logo.svg"
```
Alternatively

File: icons/piped.svg
Comment (PratyushJayachandran) [inline icons/piped.svg:1]: This doesn't seem like the original SVG @SebiTalent04!
As @adamrusted mentioned, could you try making it like this?
simpleicons
I added a 0.5px gap between the beige disks and the body in the [SVG found here.](https://github.com/TeamPiped/Piped/blob/615f1e64d72181f11d7a002b63a4d26028619230/public/img/icons/logo.svg)

File: _data/simple-icons.json [hunk 2]
```diff
@@ -9370,6 +9370,11 @@
             "hex": "1A1928",
             "source": "https://www.pioneerdj.com"
         },
+        {
+            "title": "Piped",
+            "hex": "D43225",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:9375]:
```suggestion
            "hex": "F84330",
```
Color picked from one end of the gradient. We can also do the dominant color
```suggestion
            "hex": "D83427",
```

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Please give it another try @SebiTalent04

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @SebiTalent04?

**Comment by SebiTalent04** [comment]
No, sorry. You can assign it to yourself.

File: icons/piped.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...5.33.03.68.06 1.05.1.35.06.72.13 1.1.21.38.08.77.18 1.17.28.38.13.76.27 1.14.43.38.16.76.33 1.14.52.35.2.7.42 1.04.65.32.24.63.49.93.76.28.27.54.56.79.86.23.3.44.62.62.94.17.33.31.66.43 1 .1.34.18.68.23 1.03.03.34.03.69.01 1.03-.04.34-.11.67-.21 1-.12.33-.26.64-.42.95-.19.3-.4.59-.63.88-.25.27-.52.53-.81.78-.31.24-.64.46-.97.67-.37.2-.75.38-1.13.54-.38.17-.77.32-1.16.45-.42.12-.84.22-1.25.31-.41.09-.82.17-1.21.24-.42.05-.81.09-1.19.13-.38.03-.73.06-1.06.08-.69.01-1.24.01-1.62.01h-.58s.02-.28.03-.68c0-.19.02-.41.02-.63v-.74c0-.27-.01-.56-.02-.84-.02-.31-.04-.62-.06-.9-.06-.63-.1-1.1-.1-1.1s.33.01.88.01c.26 0 .57-.02.92-.05.33-.04.68-.09 1.04-.15.34-.08.68-.18 1-.3.15-.06.3-.13.44-.2.13-.08.26-.16.38-.24.11-.09.22-.18.31-.27.09-.1.16-.2.23-.3.06-.1.1-.21.14-.32a1.242 1.242 0 0 0-.01-.66c-.04-.11-.09-.22-.15-.32a2.34 2.34 0 0 0-.24-.31c-.1-.1-.21-.19-.32-.29-.13-.09-.26-.18-.4-.26-.15-.08-.3-.15-.46-.23-.34-.13-.69-.25-1.03-.34-.37-.09-.73-.15-1.07-.21-.35-.04-.67-.07-.95-.1-.57-.03-.91-.03-.91-.03Zm.3.04c-1.06 0-1.92-1-1.92-2.24S10.68.33 11.74.33s1.92 1 1.92 2.24-.86 2.24-1.92 2.24Zm.07 9.42c-1.06 0-1.92-1.1-1.92-2.46s.86-2.46 1.92-2.46 1.92 1.1 1.92 2.46-.86 2.46-1.92 2.46Z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/piped.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Piped</title><path d="M9.915 22.79c.012.04.032.082.032.125 0 .6-1.207 1.085-2.7 1.085s-2.698-.485-2.698-1.085c0-.043.02-.084.031-.126a.584.584 0 0 1 .103-.196.996.996 0 0 1 .181-.178l.012-.009a1.71 1.71 0 0 1 .239-.15c.014-.008.03-.015.045-.022.084-.042.172-.082.27-.118.033-.012.07-.022.103-.034a3.82 3.82 0 0 1 .273-.083c.06-.015.123-.027.184-.04.083-.017.162-.036.25-.05.091-.015.19-.025.286-.036.066-.007.13-.017.197-.023a6.607 6.607 0 0 1 1.05 0c.067.006.13.016.197.023.096.011.194.02.286.036.087.014.167.033.249.05.062.013.125.025.184.04.097.025.185.054.273.083.034.012.07.022.103.034.098.036.186.076.27.118l.046.022c.09.047.168.097.239.15l.011.009a.996.996 0 0 1 .182.178.584.584 0 0 1 .102.196zm.014-.781V.79S9.593.58 9.06.33A7.336 7.336 0 0 0 8.1.07 8.257 8.257 0 0 0 6.987 0a8.05 8.05 0 0 0-1.044.144c-.305.085-.572.18-.786.272C4.776.63 4.56.79 4.56.79v21.224c.609-.463 1.668-.685 2.687-.685 1.015 0 2.072.22 2.68.68zm1.906-17.195c.052 0 .1-.017.15-.029s.001 0 .001 0a.745.745 0 0 0 .21-.087l.005-.003c.066-.04.128-.088.188-.148l.015-.016c.056-.057.108-.12.158-.192l.03-.045c.043-.067.084-.137.122-.215.014-.03.027-.063.04-.094.031-.07.061-.14.087-.217.018-.05.031-.105.046-.158.019-.066.038-.13.053-.2.017-.076.027-.157.04-.236.007-.055.018-.108.024-.164a4.137 4.137 0 0 0 0-.874c-.007-.065-.02-.124-.03-.187-.012-.074-.02-.15-.036-.22-.019-.084-.043-.161-.067-.24-.012-.041-.022-.086-.036-.126a2.584 2.584 0 0 0-.11-.264c-.009-.017-.015-.037-.024-.054-.048-.095-.1-.18-.157-.259L12.54.78c-.177-.245-.39-.395-.623-.425-.028-.004-.054-.015-.082-.015-.659 0-1.192 1-1.192 2.237s.533 2.237 1.192 2.237zm.744 8.943c.059-.089.113-.186.163-.295l.02-.049a3 3 0 0 0 .116-.303c.014-.043.024-.092.036-.137.024-.088.05-.174.068-.268.016-.078.025-.162.037-.244.01-.07.023-.136.03-.207a5.03 5.03 0 0 0 0-.969c-.007-.067-.019-.128-.027-.193-.012-.085-.022-.171-.038-.252-.017-.086-.04-.165-.061-.246-.013-.05-.025-.104-.04-.152a3.053 3.053 0 0 0-.1-.27l-.03-.076a2.234 2.234 0 0 0-.144-.267l-.013-.023c-.2-.312-.446-.504-.716-.504-.66 0-1.192 1.102-1.192 2.465 0 1.362.533 2.464 1.192 2.464.023 0 .044-.011.067-.014.236-.027.452-.191.632-.46zm6.857-6.615a12.19 12.19 0 0 0-.143-1.025 12.28 12.28 0 0 0-.27-.996 12.027 12.027 0 0 0-.387-.938 11.425 11.425 0 0 0-.492-.859c-.185-.267-.378-.52-.579-.76-.21-.232-.426-.448-.646-.65-.234-.187-.47-.358-.706-.515a9.193 9.193 0 0 0-.706-.426c-.247-.108-.49-.202-.726-.284a8.865 8.865 0 0 0-.686-.208 11.058 11.058 0 0 0-1.218-.156l-.058.001c.433.486.708 1.294.708 2.25 0 1-.3 1.836-.767 2.313l.044.007c.21.052.433.12.664.206.215.095.432.21.641.343.097.071.192.146.284.226.086.082.168.168.246.258.072.092.139.187.2.286.056.1.105.203.149.308.037.106.068.214.092.323.018.11.029.22.033.33a2.64 2.64 0 0 1-.027.326 2.66 2.66 0 0 1-.087.317c-.041.102-.089.201-.142.297-.06.095-.124.185-.194.272a3.297 3.297 0 0 1-.239.24 3.549 3.549 0 0 1-.275.205 4.3 4.3 0 0 1-.623.296 5.57 5.57 0 0 1-.644.154l-.05.004c.452.521.74 1.418.74 2.48 0 1.042-.278 1.923-.716 2.448.114-.002.22-.003.35-.007a13.912 13.912 0 0 0 1.394-.207 10.11 10.11 0 0 0 1.532-.55 8.706 8.706 0 0 0 1.422-.996c.21-.21.412-.435.604-.673a8.95 8.95 0 0 0 .505-.782 9.5 9.5 0 0 0 .39-.876c.101-.308.19-.626.262-.951.058-.329.1-.664.129-1.003.012-.34.01-.684-.008-1.028z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
@PratyushJayachandran, I'm seeing some differences in the inner edge of all 3 ovals ([example](https://github.com/simple-icons/simple-icons/assets/15157491/1e85b212-dbca-4a6b-8c5f-d20a80e2a217) from the topmost one, with yours in red). There are also a number of excess points that could be removed.

File: icons/piped.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...464 1.192 2.464.023 0 .044-.011.067-.014.236-.027.452-.191.632-.46zm6.857-6.615a12.19 12.19 0 0 0-.143-1.025 12.28 12.28 0 0 0-.27-.996 12.027 12.027 0 0 0-.387-.938 11.425 11.425 0 0 0-.492-.859c-.185-.267-.378-.52-.579-.76-.21-.232-.426-.448-.646-.65-.234-.187-.47-.358-.706-.515a9.193 9.193 0 0 0-.706-.426c-.247-.108-.49-.202-.726-.284a8.865 8.865 0 0 0-.686-.208 11.058 11.058 0 0 0-1.218-.156l-.058.001c.433.486.708 1.294.708 2.25 0 1-.3 1.836-.767 2.313l.044.007c.21.052.433.12.664.206.215.095.432.21.641.343.097.071.192.146.284.226.086.082.168.168.246.258.072.092.139.187.2.286.056.1.105.203.149.308.037.106.068.214.092.323.018.11.029.22.033.33a2.64 2.64 0 0 1-.027.326 2.66 2.66 0 0 1-.087.317c-.041.102-.089.201-.142.297-.06.095-.124.185-.194.272a3.297 3.297 0 0 1-.239.24 3.549 3.549 0 0 1-.275.205 4.3 4.3 0 0 1-.623.296 5.57 5.57 0 0 1-.644.154l-.05.004c.452.521.74 1.418.74 2.48 0 1.042-.278 1.923-.716 2.448.114-.002.22-.003.35-.007a13.912 13.912 0 0 0 1.394-.207 10.11 10.11 0 0 0 1.532-.55 8.706 8.706 0 0 0 1.422-.996c.21-.21.412-.435.604-.673a8.95 8.95 0 0 0 .505-.782 9.5 9.5 0 0 0 .39-.876c.101-.308.19-.626.262-.951.058-.329.1-.664.129-1.003.012-.34.01-.684-.008-1.028z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/piped.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Piped</title><path d="M9.9467 22.9146c0 .6002-1.2063 1.0854-2.699 1.0854s-2.699-.4852-2.699-1.0854c0-.6003 1.2064-1.0856 2.699-1.0856s2.699.4853 2.699 1.0856zm-.0179-.906V.7897s-.3356-.2092-.868-.4592C8.7843.233 8.4563.1415 8.1005.069a8.2565 8.2565 0 0 0-1.114-.0689c-.3676.027-.7237.0772-1.044.144-.3046.0851-.5716.1799-.7857.272-.381.2127-.5956.3735-.5956.3735V22.013c.6083-.4618 1.6678-.684 2.6864-.684 1.0155 0 2.072.2207 2.6811.6796zm1.9063-17.1949c.6592 0 1.1923-1.0001 1.1923-2.237 0-1.2371-.533-2.2372-1.1923-2.2372S10.643 1.3396 10.643 2.5768c0 1.2368.533 2.237 1.1922 2.237zm.0449 4.4885c-.6592 0-1.1923 1.1016-1.1923 2.4643 0 1.3627.5331 2.4643 1.1923 2.4643.6595 0 1.1923-1.1016 1.1923-2.4643 0-1.3627-.5328-2.4643-1.1923-2.4643zm7.5557-2.16a12.1903 12.1903 0 0 0-.1422-1.0257 12.2807 12.2807 0 0 0-.27-.995 12.0272 12.0272 0 0 0-.3875-.9385 11.4255 11.4255 0 0 0-.4917-.859 10.8371 10.8371 0 0 0-.5788-.7605 10.1813 10.1813 0 0 0-.6469-.6492 10.1138 10.1138 0 0 0-.7058-.5157A9.1929 9.1929 0 0 0 15.507.973a10.195 10.195 0 0 0-.7262-.2842 8.8651 8.8651 0 0 0-.6856-.208c-.2285-.0426-.4458-.076-.6498-.1015S13.0511.336 12.877.3249c-.0217.0002-.0371.001-.0583.0011.4337.4856.7088 1.2944.7088 2.2508 0 .9986-.3 1.8355-.7675 2.312.0162.003.0276.0038.044.007a6.268 6.268 0 0 1 .6641.2057c.215.0956.4316.2106.641.3438.0974.0707.1922.1458.2837.2254.086.082.1682.168.2462.258.0719.092.1387.1876.2003.2861.0555.1003.105.2034.1488.3083.0369.106.0677.2139.092.3228.0177.1094.0288.2194.033.3294a2.6392 2.6392 0 0 1-.0274.3271c-.0222.1071-.051.213-.0862.3163a2.7646 2.7646 0 0 1-.1426.2978 2.9462 2.9462 0 0 1-.1936.2716 3.2973 3.2973 0 0 1-.2388.2403 3.5486 3.5486 0 0 1-.2758.2044 4.2996 4.2996 0 0 1-.6228.296 5.5695 5.5695 0 0 1-.6439.1543c-.0187.0024-.0318.0022-.0502.0043.4528.5207.7406 1.4173.7406 2.4791 0 1.0423-.2781 1.9239-.7165 2.4485.114-.0019.2203-.003.3508-.0067.2028-.0185.4225-.0444.656-.0787a13.912 13.912 0 0 0 .7384-.1285c.2444-.0663.4958-.1449.7522-.2365.2558-.0915.5166-.196.7792-.3136a8.706 8.706 0 0 0 .7211-.4532c.238-.166.472-.347.7006-.543a8.6323 8.6323 0 0 0 .6043-.673 8.9507 8.9507 0 0 0 .5049-.7816 9.4993 9.4993 0 0 0 .3896-.876c.1015-.3086.1892-.6263.2622-.9513.0578-.329.1007-.664.1286-1.0028a11.793 11.793 0 0 0-.008-1.028z"/></svg>
```

**Comment by PratyushJayachandran** [comment]
@PeterShaggyNoble 
piped

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Give this a try, @PratyushJayachandran; managed to remove a few of the excess points - man, but the outer curve is _riddled_ with imperfections! - and get the precision down to 3. Do a quick comparison yourself, though, in case I missed anything.

```
M9.947 22.915c0 .6-1.207 1.085-2.7 1.085-1.492 0-2.698-.485-2.698-1.085s1.206-1.086 2.699-1.086c1.492 0 2.699.485 2.699 1.086zm-.018-.906V.789S9.593.58 9.06.33A7.333 7.333 0 0 0 8.1.07 8.257 8.257 0 0 0 6.986 0a8.07 8.07 0 0 0-1.043.144c-.305.085-.572.18-.786.272A5.819 5.819 0 0 0 4.56.79v21.223c.609-.462 1.668-.684 2.687-.684 1.015 0 2.072.22 2.68.68zm1.906-17.195c.66 0 1.192-1 1.192-2.237S12.494.34 11.835.34c-.66 0-1.192 1-1.192 2.237s.533 2.237 1.192 2.237zm.045 4.488c-.66 0-1.192 1.102-1.192 2.464 0 1.363.533 2.465 1.192 2.465.66 0 1.192-1.102 1.192-2.465 0-1.362-.532-2.464-1.192-2.464zm7.556-2.16a12.19 12.19 0 0 0-.142-1.026 12.28 12.28 0 0 0-.27-.994 12.027 12.027 0 0 0-.388-.939c-.151-.293-.315-.58-.492-.859a10.837 10.837 0 0 0-.578-.76 10.181 10.181 0 0 0-.647-.65 9.626 9.626 0 0 0-1.412-.941 9.61 9.61 0 0 0-1.412-.492 10.987 10.987 0 0 0-.65-.102 6.299 6.299 0 0 0-.626-.053c.433.486.708 1.294.708 2.25 0 1-.3 1.836-.767 2.313.224.041.575.163.708.212.215.096.432.211.641.344.356.258.543.471.73.77.263.474.31.856.247 1.287-.135.651-.495 1.035-.937 1.33a4.3 4.3 0 0 1-.623.297c-.087.026-.438.13-.694.158.453.521.74 1.418.74 2.48 0 1.042-.278 1.923-.716 2.448.114-.002.22-.003.35-.007.414-.037.9-.107 1.395-.207.314-.085.862-.25 1.531-.55a8.856 8.856 0 0 0 1.422-.996c.213-.214.415-.438.605-.673a8.95 8.95 0 0 0 .504-.782c.145-.285.275-.577.39-.876.102-.308.19-.626.262-.951.058-.33.1-.664.129-1.003.02-.343-.001-.686-.008-1.028z
```

File: icons/piped.svg [hunk 5]
```diff
@@ -0,0 +1 @@
... 0 0-.6856-.208c-.2285-.0426-.4458-.076-.6498-.1015S13.0511.336 12.877.3249c-.0217.0002-.0371.001-.0583.0011.4337.4856.7088 1.2944.7088 2.2508 0 .9986-.3 1.8355-.7675 2.312.0162.003.0276.0038.044.007a6.268 6.268 0 0 1 .6641.2057c.215.0956.4316.2106.641.3438.0974.0707.1922.1458.2837.2254.086.082.1682.168.2462.258.0719.092.1387.1876.2003.2861.0555.1003.105.2034.1488.3083.0369.106.0677.2139.092.3228.0177.1094.0288.2194.033.3294a2.6392 2.6392 0 0 1-.0274.3271c-.0222.1071-.051.213-.0862.3163a2.7646 2.7646 0 0 1-.1426.2978 2.9462 2.9462 0 0 1-.1936.2716 3.2973 3.2973 0 0 1-.2388.2403 3.5486 3.5486 0 0 1-.2758.2044 4.2996 4.2996 0 0 1-.6228.296 5.5695 5.5695 0 0 1-.6439.1543c-.0187.0024-.0318.0022-.0502.0043.4528.5207.7406 1.4173.7406 2.4791 0 1.0423-.2781 1.9239-.7165 2.4485.114-.0019.2203-.003.3508-.0067.2028-.0185.4225-.0444.656-.0787a13.912 13.912 0 0 0 .7384-.1285c.2444-.0663.4958-.1449.7522-.2365.2558-.0915.5166-.196.7792-.3136a8.706 8.706 0 0 0 .7211-.4532c.238-.166.472-.347.7006-.543a8.6323 8.6323 0 0 0 .6043-.673 8.9507 8.9507 0 0 0 .5049-.7816 9.4993 9.4993 0 0 0 .3896-.876c.1015-.3086.1892-.6263.2622-.9513.0578-.329.1007-.664.1286-1.0028a11.793 11.793 0 0 0-.008-1.028z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/piped.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Piped</title><path d="M9.947 22.915c0 .6-1.207 1.085-2.7 1.085-1.492 0-2.698-.485-2.698-1.085s1.206-1.086 2.699-1.086c1.492 0 2.699.485 2.699 1.086zm-.018-.906V.789S9.593.58 9.06.33A7.333 7.333 0 0 0 8.1.07 8.257 8.257 0 0 0 6.986 0a8.07 8.07 0 0 0-1.043.144 6.31 6.31 0 0 0-.786.272A5.819 5.819 0 0 0 4.56.79v21.223c.609-.462 1.668-.684 2.687-.684 1.015 0 2.072.22 2.68.68zm1.906-17.195c.66 0 1.192-1 1.192-2.237S12.494.34 11.835.34c-.66 0-1.192 1-1.192 2.237s.533 2.237 1.192 2.237zm.045 4.488c-.66 0-1.192 1.102-1.192 2.464 0 1.363.533 2.465 1.192 2.465.66 0 1.192-1.102 1.192-2.465 0-1.362-.532-2.464-1.192-2.464zm7.556-2.16a12.19 12.19 0 0 0-.142-1.026 12.28 12.28 0 0 0-.27-.994 12.027 12.027 0 0 0-.388-.939c-.151-.293-.315-.58-.492-.859a10.837 10.837 0 0 0-.578-.76 10.181 10.181 0 0 0-.647-.65 9.626 9.626 0 0 0-1.412-.941 9.61 9.61 0 0 0-1.412-.492 10.987 10.987 0 0 0-.65-.102 6.299 6.299 0 0 0-.626-.053c.433.486.708 1.294.708 2.25 0 1-.3 1.836-.767 2.313.224.041.575.163.708.212.215.096.432.211.641.344.356.258.543.471.73.77.263.474.31.856.247 1.287-.135.651-.495 1.035-.937 1.33a4.3 4.3 0 0 1-.623.297c-.087.026-.438.13-.694.158.453.521.74 1.418.74 2.48 0 1.042-.278 1.923-.716 2.448.114-.002.22-.003.35-.007.414-.037.9-.107 1.395-.207.314-.085.862-.25 1.531-.55a8.856 8.856 0 0 0 1.422-.996c.213-.214.415-.438.605-.673a8.95 8.95 0 0 0 .504-.782c.145-.285.275-.577.39-.876.102-.308.19-.626.262-.951.058-.33.1-.664.129-1.003.02-.343-.001-.686-.008-1.028z"/></svg>
```

**Review by PratyushJayachandran — COMMENTED** [review]
I applied your SVG @PeterShaggyNoble

**Review by adamrusted — APPROVED** [review]

**Pull request merged**